qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -11,6 +11,8 @@
     gagent_pkg_check_cmd = "rpm -q qemu-guest-agent"
     black_list_change_cmd = "sed -i 's/%s//g' /etc/sysconfig/qemu-ga"
     black_list_check_cmd = "grep -nr 'BLACKLIST_RPC=.*%s' /etc/sysconfig/qemu-ga"
+    black_list_backup = "/bin/cp -rf /etc/sysconfig/qemu-ga /etc/sysconfig/qemu-ga-bk"
+￼   recovery_black_list = "mv -f /etc/sysconfig/qemu-ga-bk /etc/sysconfig/qemu-ga"
```

Comment:
Better make your backup path a variable.

Reply:
okay ,editing it

Reply:
following is okay? or ..
in .py:
bl_path = "/etc/sysconfig/"
cmd_blacklist_backup = self.params["black_list_backup"] % (bl_path, bl_path)

in .cfg:
black_list_backup = "/bin/cp -rf %sqemu-ga %sqemu-ga-bk"
recovery_black_list = "mv -f %sqemu-ga-bk %sqemu-ga"

Reply:
No, I mean, you make sure the backup file is same in `black_list_backup` and `recovery_black_list`, so better define it as a variable, something like:
```
backup_file = /etc/sysconfig/qemu-ga-bk
black_list_backup = cp -f /etc/sysconfig/qemu-ga ${backup_file}
recovery_black_list = mv -f ${backup_file} /etc/sysconfig/qemu-ga

Reply:
okay,thanks,editing.